GOVERNANCE.md

```diff
@@ -28,8 +26,14 @@ No more than one-fourth of the CC members may be affiliated with the same employ
 
 The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
 
+The CC may, at its discretion, invite any number of non-voting observers to participate in the public portion of CC discussions and meetings.
+
 The CC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CC). The meeting shall be directed by the Individual Membership Directors. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
 
+CC members are expected to regularly participate in TSC activities.
```

Comment:
Would love more context around this to be added 😊 Reading this without any context, I'm quite sure what it means other than at a superficial level .